matscipy/fracture_mechanics/crack.py

```diff
@@ -907,71 +1028,91 @@ def residuals(x, mask):
                 raise RuntimeError('CLE fit failed')
         return res
 
-    def get_deformation_gradient(self,r,theta,k):
-        return self.crk.crack.deformation_gradient(r, theta, k)
+    def get_deformation_gradient(self, x, y, kI=None, kII=None, de=0):
```

Comment:
I actually do need to change the coordinate system here, as the d(shift)/d(alpha) finite differences needs access to tip_x, which is why it appears in this function. I don't think this will break anyone else's code, as I actually added this function as an exposed version of the self.crk.crack.deformation_gradient function, so I think that only I have used it so far. Likely, nothing else is calling this